bookstore-scrapper/indexTextbooks.ts

```diff
@@ -24,20 +24,40 @@ const storeDomain = "https://campusstore.mcmaster.ca";
 const storeBookPath = "/cgi-mcm/ws";
 const storeCourseMaterialPath = "/txhome.pl?wsgm=coursematerial";
 
+export enum UNWANTED_ENTRY_KEYWORD {
+  ISBN_ENDING_WITH_B = "B",
+  ETEXT = "ETEXT",
+  LAB_MANUAL = "LAB MANUAL",
```

Comment:
It looks like this enum is only referenced in `isWantedEntry`. Are you planning on using it in other files? If not, it might be better to write these strings in line within `isWantedEntry`, since it will make `isWantedEntry` easier to read.